sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -169,6 +169,8 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
       return
     }
     lifecycleState = LifecycleState.STATE_DESTROYED
+    mapboxMap.setCameraAnimationPlugin(null)
```

Comment:
Could it actually be reset in mapboxMap.onDestroy instead?

Reply:
💎